relay-server/src/actors/processor.rs

```diff
@@ -1112,7 +1113,8 @@ impl EnvelopeProcessorService {
                         }
                         Err(e) => {
                             relay_log::warn!(
-                                "replay-recording-event: failed to parse with message {}",
+                                "replay-recording-event: failed to parse {:?} with message {}",
+                                event_id,
```

Comment:
I think only `relay_log::error!` is reported to sentry: https://github.com/getsentry/relay/blob/229b7c3ddd22a2af8cbe9e14d63760ae881e76cd/relay-log/src/lib.rs#L38-L39.

Do we know how often this message would be emitted? If it's just the occasional error, then logging to sentry should be fine. If it's large bursts of data, we might have to sample down and only send every 100th message or so (e.g. by taking the replay id modulo 100).

Reply:
@jjbayer it should be in our logs though, no?

Reply:
We'll figure it out later.